server/protocols/webadmin/cli/src/mapping_entity.py

```diff
@@ -0,0 +1,38 @@
+import click
+import json
+import requests as req
+import sys
+
+
+def print_json(obj):
+    parsed = json.loads(obj.text)
+    print(json.dumps(parsed, indent=4, sort_keys=True))
+
+
+def handle_other_responses(res):
```

Comment:
I think all those utility functions (`print_json` as well) can be extracted into their own file too, then you can just import them, to avoid the repetition of those in each of your files